modules/tinymce/src/core/main/ts/api/ui/Registry.ts

```diff
@@ -16,7 +16,7 @@ const registry = (): Registry.Registry => {
      * Emoticons and Charmap use an autocompleter.
      * <br>
      * For information on creating an autocompleter, see:
-     * <a href="https://www.tiny.cloud/docs/tinymce/6/autocompleter/">
+     * <a href="https://www.tiny.cloud/docs/tinymce/7/autocompleter/">
```

Comment:
We might need to do the same for other hard coded links in the API. It would be better if the version of these links can be provided at  build time

Reply:
Thanks for doing these